src/tables.css

```diff
@@ -31,7 +31,7 @@
  *
  */
 .table,
-.prose table {
+.prose table:not(.unprose) {
```

Comment:
I would like to _either_ offer `.unprose` containers, _or_ offer `.unprose` as a single-element modifier. Giving the class both qualities feels unnecessary. What do you think about leaving this PR as-is and merging?